swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/preview/2021-09-01-preview/examples/OpenShiftClusters_CreateOrUpdate.json

```diff
@@ -6,14 +6,6 @@
     "resourceName": "resourceName",
     "parameters": {
       "location": "location",
-      "systemData": {
-        "createdBy": "string",
-        "createdByType": "Application",
-        "createdAt": "2020-02-03T01:01:01.1075056Z",
-        "lastModifiedBy": "string",
-        "lastModifiedByType": "Application",
-        "lastModifiedAt": "2020-02-03T01:01:01.1075056Z"
-      },
```

Comment:
Did this get deleted during `make client`?  Is it supposed to be deleted?  I know it's changed to a pointer but I didn't believe it would be deleted.

Reply:
See original PR link in the description